packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -300,6 +300,54 @@ export const mutation = mutationType({
       },
     })
 
+    t.liveMutation('setAutoScrollingEnabled', {
```

Comment:
It might be simpler to have a single mutation for this like `updateSettings($input: UpdateSettingsInput)`. We can change that later though

Reply:
Yeah just realized this, I want to redo the entire cache layer (for appData) and probably add this refactor then (as well as typing the rest of the appData layer etc...)